src/main/java/org/opentripplanner/openstreetmap/wayproperty/specifier/ExactMatchSpecifier.java

```diff
@@ -34,17 +34,15 @@ public ExactMatchSpecifier(String spec) {
 
   public ExactMatchSpecifier(Condition... conditions) {
     this.conditions = Arrays.asList(conditions);
-    if (this.conditions.stream().anyMatch(Condition::isWildcard)) {
```

Comment:
Do you want to use the `ExactMatchSpecifier` together with a wildcard condition or why are you removing this check?

Reply:
If you can use a `Absent` condition in ExactMatchSpecifier, i don't see any reason why you shouldn't be able to use `Present`.
I even consider to expand `parseConditions` to support parsing `!=*` as a `Absent` condition.

Reply:
Sorry for the late reply: I'm ok with this change, btw.